city-infrastructure-platform/settings.py

```diff
@@ -162,5 +164,10 @@
 # Azure CLIENT_IP middleware
 AZURE_DEPLOYMENT = env.bool("AZURE_DEPLOYMENT")
 
+# Sentry-SDK
+SENTRY_DSN = env.str("SENTRY_DSN")
+if SENTRY_DSN:
+    sentry_sdk.init(dsn=SENTRY_DSN, integrations=[DjangoIntegration()])
```

Comment:
What's your opinion on having the Git commit hash here? It could be passed as `release` keyword argument to the `init` function.

Is the release/commit information so useful that we want to keep it?

Reply:
As discussed in Slack, I'll add this release information here